community/server/src/test/java/org/neo4j/server/rest/transactional/TransactionHandleTest.java

```diff
@@ -94,6 +96,7 @@ public void shouldExecuteStatements() throws Exception
         InOrder outputOrder = inOrder( output );
         outputOrder.verify( output ).transactionCommitUri( uriScheme.txCommitUri( 1337 ) );
         outputOrder.verify( output ).statementResult( executionResult, false, (ResultDataContent[])null );
+        outputOrder.verify( output ).notifications( anyCollectionOf( Notification.class ) );
```

Comment:
Is this any better? or do we need to create something that actually verifies it is an Iterable? I like anyCollectionOf or anyListOf better than any(Iterable.class), but I guess I could create an anyIterableOf matcher